forge-gui/res/cardsfolder/upcoming/soul_partition.txt

```diff
@@ -0,0 +1,8 @@
+Name:Soul Partition
+ManaCost:1 W
+Types:Instant
+A:SP$ ChangeZone | Cost$ 2 W B | ValidTgts$ Permanent.nonLand | TgtPrompt$ Select target nonland permanent | Origin$ Battlefield | Destination$ Exile | RememberTargets$ True | SubAbility$ DBMayPlay | SpellDescription$ Exile target nonland permanent. For as long as that card remains exiled, it’s owner may play it. A spell cast by an opponent this way costs {2} more to cast.
+SVar:DBMayPlay:DB$ Effect | StaticAbilities$ STPlay | RememberObjects$ Remembered | Duration$ Permanent | ExileOnMoved$ Exile 
+SVar:STPlay:Mode$ Continuous | MayPlay$ True | EffectZone$ Command | Affected$ Card.IsRemembered+nonLand | MayPlayPlayer$ CardOwner |  AddStaticAbility$ RaiseCost |  AffectedZone$ Exile | Description$ For as long as that card remains exiled, it's owner may play it. A spell cast by an opponent this way costs {2} more to cast.
+SVar:RaiseCost:Mode$ RaiseCost | ValidCard$ Card.Self | Type$ Spell | Amount$ 2 | EffectZone$ All | Activator$ Player.Opponent
```

Comment:
I think the "opponent" should be in relation to who creates the effect (the controller of this spell)
Did you test it, I'm not sure doing it that way would apply the raise to your own cards at all?

Reply:
If i'm understanding the card correctly, it only raises cost for the opponent right ?

Reply:
sure, but the opponent based on the play permission
so depending on the case the owner will be one (otherwise the owner would be never be an opponent to itself)

Reply:
Oh so the effects needs to the property of the cardower

Reply:
The wording is really weird on this one. Thinking just removing this for now and return to it later

Reply:
The problem is you're having the exiled card "gain" the cost reduction, but that's not how that effect does it
Try like on _Elite Spellbinder_

Reply:
Thanks that made it work !